### PR TITLE
Improved notebook + metric + quick-test

### PR DESCRIPTION
**Pull request by deepcharles**
1. It is now possible to use the `--quick-test` option as in
```
ramp-test --submission <submission folder> --quick-test
```

2. The f-score is now computed on each instance (signal) and then averaged over the testing set.

3. Suggestions from #8 have been integrated to the notebook

### CODE REVIEW

**Comment by deepcharles** [comment]
Also I have reformulated some parts of data presentation to make it clear(er) that

- during a **trial**, a subject is being recorded undergoing a specific protocol with **two** sensors (one on each foot);
- each sensor produces a multivariate signal on which we apply a step detection algorithm;
- footsteps are detected per foot (which might be counter-intuitive).

Therefore, a single trial outputs two training/testing instances (left foot+right foot). However students can pool the two signals and use both sensors to detect steps, as long as they make two predictions (left foot+right foot). To pool sensors, they can group by the `.trial_code` attribute which is unique for each trial.

File: problem.py [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 import pandas as pd
-
 from rampwf.prediction_types.base import BasePrediction
 from rampwf.score_types import BaseScoreType
 from rampwf.workflows import Estimator
 from sklearn.model_selection import GroupShuffleSplit
 
 DATA_HOME = "data"
 RANDOM_STATE = 777
+random.seed(RANDOM_STATE)
```
Comment (agramfort) [inline problem.py:17]: can we rather use np.random.RandomState to avoid affecting the global state of random?

Comment (deepcharles) [reply problem.py:17]: Thanks. Solved in #15
